Pear/Models/FeedbackOption.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  FeedbackOption.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 3/9/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+struct FeedbackOption {
```

Comment:
I may be wrong, but `FeedbackOptions` seems more appropriate.

Reply:
I named it FeedbackOption since we have MenuOption and SettingOption